components/core/src/TimestampPattern.cpp

```diff
@@ -503,10 +521,56 @@ bool TimestampPattern::parse_timestamp (const string& line, epochtime_t& timesta
                     break;
                 }
 
+                case '#': {
+                    is_relative = true;
+                    break;
+                }
+
                 default:
                     return false;
             }
             is_specifier = false;
+        } else if (is_relative) {
+            int cFieldLength = 0;
```

Comment:
`cFieldLength` is the naming convention for constants, so this should really be `field_length`.

Reply:
commit 600300306a6b7c4fd725b519607f47b08f09f1a7